acbuild/acbuild.go

```diff
@@ -91,6 +93,7 @@ func init() {
 	cmdAcbuild.PersistentFlags().BoolVar(&debug, "debug", false, "Print out debug information to stderr")
 	cmdAcbuild.PersistentFlags().StringVar(&contextpath, "work-path", ".", "Path to place working files in")
 	cmdAcbuild.PersistentFlags().StringVar(&aciToModify, "modify", "", "Path to an ACI to modify (ignores build context)")
+	cmdAcbuild.PersistentFlags().BoolVar(&disableTracking, "no-tracking", false, "Don't add annotations with the command that was run")
```

Comment:
I like `--no-history` better than `--no-tracking` at the very least, so I switched to that.